exporter/postgres/setup_pg96.sql

```diff
@@ -1,7 +1,11 @@
 DO $$
 BEGIN
+    IF NOT EXISTS (SELECT 1 FROM pg_roles WHERE rolname = 'pgmonitor') THEN
+        CREATE ROLE pgmonitor;
```

Comment:
When upgrading to PG >= 10 we will need cleanup. Please add.

Reply:
Updated and included some comments, thanks.